app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,37 @@
+class PostsController < ApplicationController
+  expose_decorated :post
+
+  before_action :authorize_resource, only: %i[new edit create update]
+
+  def new
+  end
+
+  def edit
+  end
+
+  def create
+    respond_with create_post
```

Comment:
I think that using interactor for such simple actions is unnecessary.
```suggestion
def create
  post.user = current_user
  post.save
  
  respond_with post
end
```